trae_api/core/resources.py

```
@@ -0,0 +1,303 @@
+            
+            try:
+                # Record request size if we have metrics and request
+                if metrics and request:
+                    try:
+                        # Try to get request body size
+                        if hasattr(kwargs.get('request'), 'model_dump_json'):
+                            request_size = len(
+                                kwargs['request'].model_dump_json().encode('utf-8')
+                            )
+                            metrics.record_request_size(request_size)
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)